mne/realtime/epochs.py

```diff
@@ -314,6 +314,7 @@ def _process_raw_buffer(self, raw_buffer):
         verbose = 'ERROR'
         sfreq = self.info['sfreq']
         n_samp = len(self._raw_times)
+        prev_samp = int(self._find_events_kwargs['min_samples'] - 1)
```

Comment:
This seems to have worked, but if the trigger channel is high at the beginning of the whole file it doesn't still work because of this if https://github.com/mne-tools/mne-python/blob/master/mne/realtime/epochs.py#L347. The last test is still failing (multiple events at the buffer boundary)